src/SectionTitle.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import cx from 'classnames';
+import Chunk from './Chunk';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+export const SECTIONTITLE_CLASS = 'sectionTitle';
+/**
+ * @module SectionTitle
+ */
+class SectionTitle extends React.Component {
+	render() {
+		const {
+			action,
+			className,
+			title,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			SECTIONTITLE_CLASS,
+			className
+		);
+
+		return (
+			<Flex
+				align='center'
+				className={classNames}
+				{...other}
+				>
+				<FlexItem>
```

Comment:
For this component, I don't feel strongly about moving away from `Chunk`. It was a typical pattern in Sassquatch2 to put chunks inside of row items (rather than around them). It makes sense to me, but if it gets weird for other people, I'm on board with `margin--bottom` classes. 

The nice thing is that `SectionTitle` abstracts it away either way.